Screenbox/Screenbox.csproj

```diff
@@ -461,52 +461,76 @@
       <LastGenOutput>Resources.generated.cs</LastGenOutput>
       <SubType>Designer</SubType>
     </PRIResource>
+    <PRIResource Include="Strings\ar-SA\KeyboardResources.resw" />
```

Comment:
Oh we were missing the keyboard resources huh

Reply:
🙄 My bad, I thought that it was automatic with the Crowdin action

Reply:
I just realized, what's stopping people from not following the keyboard constraints? For example, you can enter more than one character for a resource that requires exactly one capital letter.

Reply:
I think [Crowdin](https://support.crowdin.com/string-management/#string-editing) provires some assistance there.

Reply:
> I just realized, what's stopping people from not following the keyboard constraints? For example, you can enter more than one character for a resource that requires exactly one capital letter.

So, Crowdin does have some [syntax](https://support.crowdin.com/expression-syntax-elements/) support, but I don't think [.resw](https://store.crowdin.com/resw) supports it.
If I'm reading this [right](https://support.crowdin.com/string-management/#adding-strings) correctly, only formats that allow editing of the source files are supported (eg. [resx](https://store.crowdin.com/resx)). Maybe it's only a limitation on adding and editing strings, but it still provides validation.

Reply:
Without Crowdin, we have no way of enforcing these rules, right? Many contributors prefer creating a PR and editing the .resw files directly. This is not a major concern since Crowdin still has precedence over the resource files; it's just something to keep in mind.